src/flows/components/panel/FlowPanel.tsx

```diff
@@ -108,6 +120,61 @@ const FlowPanel: FC<Props> = ({id, controls, persistentControls, children}) => {
     /* eslint-enable no-console */
   }, [id])
 
+  const [size, updateSize] = useState<number>(
+    flow.meta.get(id).height || DEFAULT_RESIZER_HEIGHT
+  )
+  const [isDragging, setIsDragging] = useState(false)
+  const bodyRef = useRef<HTMLDivElement>(null)
+  const handleRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (isDragging === true) {
+      handleRef.current &&
+        handleRef.current.classList.add('panel-resizer--drag-handle__dragging')
+    }
+
+    if (isDragging === false) {
+      handleRef.current &&
+        handleRef.current.classList.remove(
+          'panel-resizer--drag-handle__dragging'
+        )
+      flow.meta.update(id, {height: size})
+    }
+  }, [isDragging]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  const handleMouseMove = (e: MouseEvent): void => {
+    if (!bodyRef.current) {
+      return
+    }
+
+    const {pageY} = e
+    const {top} = bodyRef.current.getBoundingClientRect()
+
+    const updatedHeight = Math.round(
+      Math.max(pageY - top, MINIMUM_RESIZER_HEIGHT)
+    )
+
+    updateSize(updatedHeight)
+  }
+
+  const handleMouseDown = (): void => {
+    setIsDragging(true)
+    const body = document.getElementsByTagName('body')[0]
+    body && body.classList.add('panel-resizer-dragging')
+
+    window.addEventListener('mousemove', handleMouseMove)
```

Comment:
do we want metrics on how often this is used?

Reply:
def not how muched it moved, but idk how much i care if someone dragged something around right now